src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.DiagnosticState.cs

```diff
@@ -16,303 +16,305 @@
 
 namespace Microsoft.CodeAnalysis.Diagnostics.EngineV1
```

Comment:
no idea why this is happening. but I see many times just synced file has messed up formatting...